source/frontend/src/features/properties/map/MapContainer.tsx

```diff
@@ -75,7 +74,7 @@ const MapContainer: React.FC<React.PropsWithChildren<MapContainerProps>> = () =>
         <CompensationRequisitionRouter setShowActionBar={setShowActionBar} />
         <PropertyActivityRouter setShowActionBar={setShowActionBar} />
       </SideBarContextProvider>
-      {!(showActionBar || mapSideBarViewState?.isFullWidth) && (
```

Comment:
@FuriousLlama can you please validate this? I wasn't able to see the base layer toggle after removing this, but I don't know the exact repro steps.